airbyte/_executors/util.py

```diff
@@ -282,7 +282,7 @@ def get_connector_executor(  # noqa: PLR0912, PLR0913, PLR0914, PLR0915, C901 #
         host_temp_dir = TEMP_DIR_OVERRIDE or Path(tempfile.gettempdir())
         container_temp_dir = DEFAULT_AIRBYTE_CONTAINER_TEMP_DIR
 
-        local_mount_dir = Path().absolute() / name
+        local_mount_dir = Path().absolute() / f"{DEFAULT_LOCAL_MOUNT_PREFIX}_{name}"
```

Comment:
This feels wrong. Revert the 'DEFAULT_LOCAL_MOUNT_PREFIX' constant-related changes.

Reply:
Actually - maybe if anything, this should be relative to project dir?